src/lib/bootloader/grub2_widgets.rb

```diff
@@ -388,9 +390,11 @@ def validate
 
       Yast::Popup.ContinueCancel(
         _(
-          "The new secure-boot enabled boot data format works only on z15 " \
-          "and later and only for zFCP disks.\n\n" \
-          "The system does not boot if these requirements are not met."
+          "The secure boot IPL works only on IBM z15, IBM LinuxONE III or later.\n" \
+          "NVMe disks works from IBM LinuxONE III.\n" \
+          "FC-attached SCSI disks needs at least IBM LinuxONE III or a IBM z15.\n" \
+          "ECKD DASDs with CDL layout works on IBM z16, LinuxONE 4 or newer.\n" \
+          "If these requirements are not met, the system will not IPL in secure mode."
```

Comment:
And here, what about something like this?

```suggestion
          "The secure boot IPL works only on IBM z15, IBM LinuxONE III or later.\n" \
          "And also note the following restrictions:" \
          "NVMe disk works only from IBM LinuxONE III.\n" \
          "FC-attached SCSI disk needs at least IBM LinuxONE III or a IBM z15.\n" \
          "ECKD DASD with CDL layout works on IBM z16, LinuxONE 4 or newer.\n" \
          "If these requirements are not met, then the system will not boot in\n" \
          "secure mode."
```